spec/unit/util/windows/adsi_spec.rb

```diff
@@ -41,6 +41,23 @@
     end
   end
 
+  describe ".domain_role" do
+    DOMAIN_ROLES = if defined? Puppet::Util::Windows::ADSI::User::DOMAIN_ROLES
```

Comment:
Travis specs would fail in a weird way without this: https://travis-ci.org/puppetlabs/puppet/jobs/620115891

Reply:
Ahhh ok, I (think) I know why. It's the rspec double parsing.  That class is not required on non-Windows platforms therefore that constant doesn't exist in that rspec context.

An alternate (not saying it should be used) would be to use `Puppet::Util::Platform.windows?` like on the describe block in line 5

Reply:
changed check to use `Puppet::Util::Platform.windows?`, also one-lined it with the ternary operator